docs/guides/tutorials/explore_expectations_in_a_notebook.rst

```diff
@@ -60,6 +62,18 @@ All of these steps take place within your notebook:
         )
 
     This method will convert your boring old pandas ``DataFrame`` into a new and exciting great_expectations ``PandasDataset``. The two classes are absolutely identical, except that ``PandasDataset`` has access to Great Expectations' methods.
+    
+    - a ``Spark DataFrame``, you can use ``SparkDFDataset``:
+    
+    .. code-block:: python
+    
+        from great_expectations.dataset.sparkdf_dataset import SparkDFDataset
+    
+        my_df = SparkDFDataset(my_spark_dataframe)
+    
+    This method will convert your ``Spark DataFrame`` into a class with access to Great Expectations' methods, such as ``ProfilingResultsPageRenderer``.
```

Comment:
```suggestion
    This method will create an object with access to Great Expectations' methods, such as ``ProfilingResultsPageRenderer``.
```

Reply:
The "convert" language is correct for Pandas, but not precise for Spark DataFrames.